dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -740,19 +740,13 @@ enum LastAction {
         if (!initialStartupCompleted) {
             // make sure pool size during startup is not greater than maxThreads
             setPoolSize(Math.min(startupPoolSize, this.maxThreads));
-            // controller cycle only needs to run during startup to monitor for possible hang condition,
-            // and checking for hang is only useful if increasing the pool size is permitted
-            if (startupPoolSize < maxThreads) {
-                activeTask = new IntervalTask(this);
-                timer.schedule(activeTask, interval, interval);
-            }
         } else {
             setPoolSize(coreThreads);
-            if (coreThreads != maxThreads) {
-                // start controller cycle
-                activeTask = new IntervalTask(this);
-                timer.schedule(activeTask, interval, interval);
-            }
+        }
+        // The controller cycle will always run, even if (coreThreads == maxThreads), to enable trace to show the pool stats
+        if (activeTask == null && !paused) {
+            activeTask = new IntervalTask(this);
+            timer.schedule(activeTask, interval, interval);
```

Comment:
```suggestion
        }
        // The controller cycle will always run, even if (coreThreads == maxThreads), to enable trace to show the pool stats
        activeTask = new IntervalTask(this);
        timer.schedule(activeTask, interval, interval);
```
No need to have the if statement.  Also will need to remove the `}` after this block.

Reply:
Done